scan/lib/scan/report_collector.rb

```diff
@@ -38,7 +38,12 @@ def generate_commands(path, types: nil, output_file_name: nil)
           next
         end
 
-        file_name = "report.#{type}"
+        case type
+        when "junit"
+          file_name = "TEST-report.xml"
```

Comment:
Tipycally name for junit (Surefire Test Report) report is TEST-[report name].xml so I think the best choose is use report as "report name" part.